src/introduction.md

```diff
@@ -139,9 +139,9 @@ attention to making those sections the best that they can be.
 [_Expression_]: expressions.md
 [cargo book]: ../cargo/index.html
 [cargo reference]: ../cargo/reference/index.html
-[expressions chapter]: expressions.md
-[lifetime of temporaries]: expressions.md#temporary-lifetimes
-[linkage]: linkage.md
+[expressions chapter]: expressions.html
```

Comment:
These md -> html changes may not really useful as mdbook handles that. cc @ehuss

Reply:
@lzutao can you send a PR to fix it?